### PR TITLE
Fix Ironic installation link

### PR DESCRIPTION
**Pull request by SafeEHA**
While trying to squash commits in #326, I ran into issues with git rebase in my local environment as I stated in the last comment I made in #326 before updating my fork

I also noticed that #323 has been merged so I only need this to be merged if there are no more issues.

Thank you

cc: @tuminoid @elfosardo

### CODE REVIEW

**Review by tuminoid — APPROVED** [review]
LGTM

File: docs/user-guide/src/capm3/installation_guide.md
Comment (lentzi90) [inline docs/user-guide/src/capm3/installation_guide.md:14]: I think it would be better to do a relative link here instead. Then it will be correct also if the domain changes (e.g. in previews generated for each PR).
```suggestion
1. Install Ironic, refer to [this page](../ironic/ironic_installation.html).
```

**Review by lentzi90 — COMMENTED** [review]
Thank you for the fix!
Could you please squash the commits into 1? With that this is good to go! :slightly_smiling_face:

**Comment by SafeEHA** [comment]
Thanks @lentzi90 
I have squashed the commits to one, I believe we can go ahead and merge

**Comment by SafeEHA** [comment]
I would also like to add that the link you @lentzi90 suggested returned an error after I made the first commit so I had to revert to the initial link so we don't run into any more problems
Thanks

**Comment by lentzi90** [comment]
> I would also like to add that the link you @lentzi90 suggested returned an error after I made the first commit so I had to revert to the initial link so we don't run into any more problems Thanks

Where did you see this error? I explicitly checked it in the preview generated by netlify and it worked great! :thinking:

**Comment by SafeEHA** [comment]
> > I would also like to add that the link you @lentzi90 suggested returned an error after I made the first commit so I had to revert to the initial link so we don't run into any more problems Thanks
> 
> Where did you see this error? I explicitly checked it in the preview generated by netlify and it worked great! thinking

I saw the error when I checked here on github, not on the netlify preview... i have resolved it now with the relative link and it works so i believe this PR can be merged and closed.

Thanks

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Comment by SafeEHA** [comment]
Hello, 
please is there anything left for me to do that I missed?

**Comment by tuminoid** [comment]
> Hello,
> please is there anything left for me to do that I missed?

It needs approval from one of the approvers in userguide OWNERS file.

**Comment by SafeEHA** [comment]
I understand, Thanks @tuminoid

**Review by lentzi90 — COMMENTED** [review]
/approve

**Comment by lentzi90** [comment]
Ah no approve rights here :slightly_frowning_face: 
Thanks for the contribution!

**Pull request merged**
